speedy-recovery-client/src/service/FhirDataQueryingService.js

```diff
@@ -0,0 +1,23 @@
+/*global FHIR */

```

Comment:
Please delete lines 1+2 and instead `import FHIR from "fhirclient/fhir-client";` - pretty cool, wasn't sure if that would work, but it does. :)

Reply:
I changed this code but it dosen't work, so I keeped the old one